client/src/components/Dashboard/CreateNewRelay/CreateNewRelay.js

```diff
@@ -1,5 +1,85 @@
 import React from 'react';
+import {
+  Form, Input, Row, Col, Button, Container, Label,
+} from 'reactstrap';
+import Header from '../../common';
+import ArrowRight from '../../../assets/icons/angle-right-solid.svg';
+import ArrowDown from '../../../assets/icons/angle-down-solid.svg';
 
-const CreateNewRelay = () => <h1>CreateNewRelay</h1>;
+const mobileScreen = window.innerWidth <= 425;
+
+const CreateNewRelay = () => {
+  return (
+    <>
+      <Header />
+      <Container>
+        <Form>
+          <Row>
```

Comment:
you dont need row component if you have single col, you can replace the Row/Col pair with div and then add classes to the div.